### ISSUE TITLE
Add support for name parameter

### ISSUE THREAD

**Issue by mateuszdrab** [dgshue/home-assistant-custom-components#2]
First of all, thank you for creating this component, it is exactly what I was looking for and I am already all set up to use it.

I'd like to suggest that support for using the name parameter is added, in my case I only import a single rule from PFsense and the generated name is rather long due to the informative comment. I've modified the script a bit to make it respect the name parameter and only import the first match using that name, but ideally, the script should find all rules for the prefix, if there's one of them, apply the provided name, if there are more matches, use that name as a prefix and add _1 after. However, I can see that the most useful use case would be to import a single rule and give it a different name in PFsense which is what my modification does. If more than one match is found, the name parameter is ignored.

I'm happy to open a pull request for that if you like.

Regards

**Comment by dgshue** [comment]
Yeah, do a pull request and let me take a look. Thanks for the contribution.

**Comment by burntoc** [comment]
> First of all, thank you for creating this component, it is exactly what I was looking for and I am already all set up to use it.
> 
> I'd like to suggest that support for using the name parameter is added, in my case I only import a single rule from PFsense and the generated name is rather long due to the informative comment. I've modified the script a bit to make it respect the name parameter and only import the first match using that name, but ideally, the script should find all rules for the prefix, if there's one of them, apply the provided name, if there are more matches, use that name as a prefix and add _1 after. However, I can see that the most useful use case would be to import a single rule and give it a different name in PFsense which is what my modification does. If more than one match is found, the name parameter is ignored.
> 
> I'm happy to open a pull request for that if you like.
> 
> Regards

I think I have this set up right - at least FauxAPI looks right and HA config checks out good, but I'm a HA newb and I'm trying to figure out next steps.  Sounds like you have a handle on it.  I can't figure out how to message you. but is there a way we can connect and would you be willing to help with the basic way to get it in the GUI and use it?  Your enhancement sounds great, btw.

**Comment by travisghansen** [comment]
https://github.com/travisghansen/hass-pfsense
